sql/firefox_accounts_derived/fxa_log_device_command_events_v1/query.sql

```diff
@@ -0,0 +1,48 @@
+WITH hmac_key AS (
+  SELECT
+    AEAD.DECRYPT_BYTES(
+      (SELECT keyset FROM `moz-fx-dataops-secrets.airflow_query_keys.fxa_prod`),
+      ciphertext,
+      CAST(key_id AS BYTES)
+    ) AS value
+  FROM
+    `moz-fx-data-shared-prod.firefox_accounts_derived.encrypted_keys_v1`
+  WHERE
+    key_id = 'fxa_hmac_prod'
+)
+--
+SELECT
+  `timestamp`,
+  jsonPayload.type,
+  TO_HEX(
+    udf.hmac_sha256((SELECT * FROM hmac_key), CAST(jsonPayload.fields.uid AS BYTES))
+  ) AS user_id,
+  TO_HEX(
+    udf.hmac_sha256(
+      (SELECT * FROM hmac_key),
+      CAST(FORMAT('%d', CAST(jsonPayload.fields.index AS INT64)) AS BYTES)
+    )
+  ) AS index,
```

Comment:
`index` required some additional logic because it's represented as a float in the BQ table. It appears these are all really integers, so we cast to int, format that int as a string, and then cast the unicode string to bytes to send through the hmac hash. As long as the assumption that these are logically integers is sound, then this should give a stable hash for different integers.